src/store/chat/saga.ts

```diff
@@ -82,6 +85,25 @@ export function* setActiveConversation(id: string) {
   history.push({ pathname: `/conversation/${id}` });
 }
 
+export function* performValidateActiveConversation() {
+  yield put(setIsErrorDialogOpen(false));
+
+  const conversationList = yield select(rawConversationsList());
+  const activeConversationId = yield select(activeConversationIdSelector);
+  const isMemberOfActiveConversation = conversationList.some((c) => c === activeConversationId);
+  if (!activeConversationId) {
+    return;
+  }
+
+  if (activeConversationId && !isMemberOfActiveConversation) {
+    yield put(setIsErrorDialogOpen(true));
+  }
+}
+
+export function* closeErrorDialog() {
+  yield call(openFirstConversation);
```

Comment:
We should also `setIsErrorDialogOpen` to false, right? That's what closes the dialog.